lib/views/widgets/tabs/tab_buttons.dart

```diff
@@ -23,29 +21,26 @@ class TabButtons extends StatelessWidget {
           25.0,
         ),
       ),
-      child: Expanded(
-        child: Padding(
-          padding: const EdgeInsets.all(5),
-          child: TabBar(
-            controller: tabController,
-            indicator: BoxDecoration(
+      child: Padding(
+        padding: const EdgeInsets.all(5),
+        child: TabBar(
+          controller: tabController,
+          indicator: BoxDecoration(
               borderRadius: BorderRadius.circular(
                 25.0,
               ),
-              color: index == 0 ? AppColor.pink : AppColor.darkBlue,
+              color: index == 0 ? AppColor.pink : AppColor.darkBlue),
```

Comment:
to trigger formatting
```suggestion
              color: index == 0 ? AppColor.pink : AppColor.darkBlue,),
```